packages/Core/theme/tags.ts

```diff
@@ -107,26 +101,58 @@ export const getTags = (theme: WuiTheme): ThemeTags => {
       6: { ...withoutVisibleBorder(colors.sub[6]) },
       7: { ...withoutVisibleBorder(colors.sub[7]) },
     },
+    hover: {
+      default: {
+        borderColor: colors.nude[600],
+      },
+      primary: {},
+      secondary: {
+        borderColor: colors.dark[400],
+      },
+      success: {
+        borderColor: colors.success[500],
+      },
+      error: {
+        borderColor: colors.danger[500],
+      },
+      warning: {
+        borderColor: colors.warning[500],
+      },
+      info: {
+        borderColor: colors.info[500],
+      },
+      1: {},
+      2: {},
+      3: {},
+      4: {},
+      5: {},
+      6: {},
+      7: {},
```

Comment:
i think we need also hover on this variants

Reply:
For primary and subs, we use [withoutVisibleBorder()](https://github.com/WTTJ/welcome-ui/pull/1503/files#diff-8c6efd32613876e4549318477a7d44c7ab5e0de209b90e0add8c367514d0b4fcR65). Do we need to add `borderColor` on hover ?

Reply:
We i think a dark.200 for example to get the same things on subs like the others on hover